### DISCUSSION TITLE
How to always activate branch

### DISCUSSION THREAD

**Discussion by lcrockett** [nvim-lualine/lualine.nvim#982] [Q&A] [answered]
I'd like the branch section to always be active. If i'm not in an git repo i'd like to show 'NA'. If i'm in a git repo i'd like to show the branch name, per default behaviour of the branch component. Is this possible with the current Lualine implementation ? For LSP diagnostics there's a context switch to set it to `always_visible`, however no such switch exists currently for `branch`.

I've tried playing around with `cond` and ensuring it always returns `true`, to no avail unfortunately. Any hints are welcome.

**Comment by fitrh** [comment, marked as answer]
I think you can use `fmt` component options 


```lua 
{
    'branch',
    fmt = function (str)
        if str == '' then 
            return 'NA'
        end 
        return str 
    end,
}
```

**Reply by lcrockett** [reply]
Of course, that worked like a charmadillo ! Cheers
